src/__test__/Details.test.js

```diff
@@ -0,0 +1,44 @@
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import configureStore from 'redux-mock-store';
+import DisplayEpisodes from '../components/DisplayEpisodes';
+import store from '../redux/store';
+
+const mockStore = configureStore([]);
+
+describe('anime redux state tests', () => {
+  it('expects to initially set anime as an empty object', () => {
+    const state = store.getState().animeList;
+    const initialState = {
+      animeList: [],
+      loading: false,
+    };
+    expect(state).toEqual(initialState);
+  });
+  it('expects to display one anime episode from mock store', () => {
+    const episodes = [
```

Comment:
as required you should test for several components in your app but you have only tested your details page I suggest you test your home page as well and test some pure functions as well 😄